script.js

```diff
@@ -0,0 +1,22 @@
+const mainMenu = document.querySelector('.nav-bar');
+const openMenu = document.querySelector('.humburger');
+const closeMenu = document.querySelector('.close-menu');
+const menuItems = document.querySelectorAll('.nav-link');
+
+function show() {
+  mainMenu.style.display = 'flex';
+  mainMenu.style.top = '0';
+}
+
+function close() {
+  mainMenu.style.top = '-100%';
+}
+
+openMenu.addEventListener('click', show);
+closeMenu.addEventListener('click', close);
+
+menuItems.forEach((item) => {
```

Comment:
- [ ] You have done great so far but there are a few changes you need to make before moving to the next milestone. :+1: :+1: Looking at the screenshot below, you can see that your mobile menu doesn't cover your full screen. There is a white space at the right. According to the fgma design, the mobile menu should cover the full screen, I'll suggest you fix this. :+1: 

- - YOUR DESIGN:
![Selection_070](https://github.com/albertkantwi/Portfolio_Project/assets/107632780/6f23eabd-5404-44b3-8b0f-f4cf0b438da7)

- - FIGMA DESIGN:
![Screenshot from 2023-06-21 19-14-42](https://github.com/albertkantwi/Portfolio_Project/assets/107632780/d136b8d2-f5d8-439d-a106-a2f23be18d8c)

> You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
- [ ] Also, as stated in the project requirement, every element should look like the Figma design as much as possible both in color and size. Currently, your close button color is a bit different from the Figma design, I'll suggest you update this. :+1: :smile: 

- - Your design:
![Selection_071](https://github.com/albertkantwi/Portfolio_Project/assets/107632780/f92a8c1c-9b00-41bf-9273-5af8e91087c4)

- - Figma design: 
![Screenshot from 2023-06-21 19-14-56](https://github.com/albertkantwi/Portfolio_Project/assets/107632780/bbe95816-1e41-477b-b345-44befac56803)